core/test/src-2.13/peschke/UnitSpec.scala

```diff
@@ -11,8 +11,6 @@ import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
 
 trait CommonSpec extends Matchers with EitherValues with OptionValues
 
-trait UnitSpec extends AnyWordSpec with CommonSpec
```

Comment:
nit: `UnitTestSpec`?

**`[not-an-action-item]`** **`[your-thoughts?]`**

Reply:
Wasn't needed, all the tests in this module are `PropSpec` tests, so it let me trim an additional dependency.

Reply:
Also, because `*Spec` classes are all tests, I didn't include "Test" in the name (felt redundant)